client/src/common/components/ArchivedMessage.jsx

```diff
@@ -69,8 +83,14 @@ class ArchivedMessage extends PureComponent {
           <Dialog
             title={
```

Comment:
change to:
`title={formatMessage({ id: pending ? 'common.archived-message.deleting' : 'common.archived-message.perm-delete' }, {name, item})}`